flink-runtime-web/web-dashboard/app/scripts/index.coffee

```diff
@@ -29,8 +29,8 @@ angular.module('flinkApp', ['ui.router', 'angularMoment'])
 # --------------------------------------
 
 .value 'flinkConfig', {
-  jobServer: ''
-# jobServer: 'http://localhost:8081/'
+#  jobServer: ''
+  jobServer: 'http://localhost:8081/'
```

Comment:
That was for local testing. Very good catch. Otherwise the web frontend build would have been broken.